src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppAppRunner/AppStack.cs

```diff
@@ -19,9 +19,13 @@ namespace AspNetAppAppRunner
 {
     public class AppStack : Stack
     {
+        private readonly Configuration _configuration;
```

Comment:
There is no usage of _configuration, is this only to let people there is configuration object?

Reply:
Yes, if a user wants access to the configuration in the customization they have to know to how to capture during the constructor. I want to make that easy for users to find the settings during the customization callback. 

It was an awkward part of my recent demo.